### ISSUE TITLE
Review-2020.6.8 ~ 2020.6.14

### ISSUE THREAD

**Issue by SuyueLiu** [SuyueLiu/dissertation-diary#2]
# TODO 

- [ ] YOLO V3源码构建

- [x] 阅读ssd并总结重点内容

- [x] 莫烦Pytorch第四章学习

- [ ] 学习NNDL第四章

# Review
* 读完了SSD的时候觉得较为轻松，能够理解论文主旨和细节，但是不清楚具体的实现细节。
* 开始跟着https://github.com/eriklindernoren/PyTorch-YOLOv3 复现yolo v3，但是里面有太多的细节值得深究了，下周继续复现。
* 看完了莫烦的Pytorch教程，抽空重复看几次，里面的一些画图方法值得学习。

**Comment by SuyueLiu** [comment]
Batch Normalization是如何工作的？BN为什么能够优化NN的训练？
- https://zhuanlan.zhihu.com/p/34879333
